bn-apps/memberships-management/membership-service-contracts-and-states/src/resources/migration/Schema.changelog-PendingMembershipRequestSchemaV1.xml

```diff
@@ -0,0 +1,10 @@
+<changeSet author="R3 Corda" id=" create PersistentPendingMembershipRequest table">
+    <createTable tableName="pending_membership_requests">
+        <column name="id" type="INT">

```

Comment:
This value needs to be autogenerated. Something along these lines: https://www.liquibase.org/documentation/changes/add_auto_increment.html